commands/configure/index.js

```diff
@@ -95,14 +112,14 @@ const setSkippedExperience = (api, application) => {
 program
   .description('Configures and associates a directory on disk to represent one of your Losant applications and its resources.')
   .action(async (command) => {
-    const userConfig = await loadUserConfig() || {};
-    if (!userConfig.apiToken) {
-      return logError('Must run losant login before running losant configure.');
-    }
+    let userConfig = await loadUserConfig() || {};
     await Promise.all(DIRECTORIES_TO_GENERATE.map((dir) => { return ensureDir(dir); }));
     await Promise.all(LOCAL_META_FILES.map((type) => { return saveLocalMeta(type, {}); }));
+    const url = await getApiURL(userConfig);
```

Comment:
at this point if URL is not set the user would need to run losant login, so I might change your getAPIUrl and returrn null when you log an error then if url is not set here, you need to return from this function.

Reply:
so, in getApiUrl I have 
```  if (keys.length < 1) {
    return logError('Must run losant login before running losant configure.');
  }
```
and even then, configure errors from the missing credentials before it reaches this point.
Unless I'm missing something, I don't think this is necessary? If url is not set at this point, configure would have errored already, right?